src/ripple/app/tx/impl/Transactor.h

```diff
@@ -51,6 +51,66 @@ struct PreflightContext
     }
 };
 
+struct PreflightResult
+{
+public:
+    PreflightContext const ctx;
+    TER const ter;
+
+    PreflightResult(PreflightContext const& ctx_,
+        TER ter_)
+        : ctx(ctx_)
+        , ter(ter_)
+    {
+    }
+};
+
+/** State information when determining if a tx is likely to claim a fee. */
+struct PreclaimContext
+{
+public:
+    Application& app;
+    ReadView const& view;
+    TER const preflightResult;
```

Comment:
Making the struct `const` when there are non-`const` reference members leaves those members mutable.

http://melpon.org/wandbox/permlink/HiOdukqWRT1lOvn1